db/seeds.rb

```diff
@@ -5,3 +5,30 @@
 #
 #   movies = Movie.create([{ name: 'Star Wars' }, { name: 'Lord of the Rings' }])
 #   Character.create(name: 'Luke', movie: movies.first)
```

Comment:
**Change request:** Delete the comments

Reply:
I've dropped the example seeds from the comments but I think the information about how the seeds file works is worth keeping, seeing as there are a limited number of seeds there at the moment.

Reply:
👍